pandas/tests/test_common.py

```diff
@@ -245,3 +246,20 @@ def test_str_size():
     ]
     result = subprocess.check_output(call).decode()[-4:-1].strip("\n")
     assert int(result) == int(expected)
+
+
+def test_bz2_missing_import():
```

Comment:
Could you add `@pytest.mark.single_cpu` to this test?

Reply:
Done.